Pipfile

```diff
@@ -9,15 +9,14 @@ fastapi = "*"
 uvicorn = "*"
 click = "*"
 boto3 = "*"
-beautifulsoup4 = "*"
-gffutils = "*"
-requests = "*"
-"biocommons.seqrepo" = "*"
-"ga4gh.vrs" = {version = ">=0.7.5.dev1", extras = ["extras"]}
-"ga4gh.vrsatile.pydantic" = ">=0.0.10"
+"ga4gh.vrsatile.pydantic" = "==0.0.11"
 
 [dev-packages]
 gene = {editable = true, path = "."}
+gffutils = "*"
+"biocommons.seqrepo" = "*"
+"ga4gh.vrs" = {version = "==0.8.0dev0", extras = ["extras"]}
+psycopg2-binary = "*"
```

Comment:
I don't know if we have a way around this with Pipenv, but I wish that the ETL dependencies were a separate class of requirements (a la extras in VRS Python) rather than "dev" (you could pretty conceivably want to build the DB yourself without pulling in pytest and jupyterlab)

Reply:
Oh, do you want me to rename to etl?

Reply:
And separate them out? from dev vs etl?

Reply:
Oh sorry, I thought you commented on the setup.cfg. Let me know if you want me to rename `dev` to `etl` or if it's fine to stick with `dev`.  @jsstevenson

Reply:
I think if we were just maintaining a setup.cfg file it'd be best to provide the ETL dependencies as a separate group, but since we straddle reqs.txt/setup.cfg/Pipfile, there's probably not a consistent way to retain those groupings, so it's fine as it is.